packages/client/src/logic/Timer.ts

```diff
@@ -0,0 +1,57 @@
+import { Game } from './Game';
+import { GameEntities } from './types';
+
+export class Timer {
+    private ctx: CanvasRenderingContext2D;
+    private canvasHeight: number;
+    private canvasWidth: number;
+    private gameRoundDuration: number | undefined;
+    private gameStartMoment: Date | undefined;
+    private gameEndMoment: Date | undefined;
+    private gameTimer: number | undefined;
+    private game: Game;
+    private now: number | undefined;
+    public then: number | undefined;
+    private elapsedTimer: number | undefined;
+
+    constructor(
+        ctx: CanvasRenderingContext2D,
+        canvasHeight: number,
+        canvasWidth: number,
+        { game }: GameEntities
+      ) {
+        this.ctx = ctx;
+        this.canvasHeight = canvasHeight;
+        this.canvasWidth = canvasWidth;
+        this.game = game;
+        this.initGameTimer();
+      }
+
+    initGameTimer(){
+    this.gameRoundDuration = 1*60000; //the reason we multiply minutes by 60000 is to convert minutes to milliseconds
```

Comment:
Что бы не писать коммент, можно просто:
```suggestion
    this.gameRoundDuration = 1 * 60 * 1000; //the reason we multiply minutes by 60000 is to convert minutes to milliseconds
```